.github/workflows/create_automerge_pr.yml

```diff
@@ -102,7 +118,13 @@ jobs:
 
           gh pr create \
             --base "${{ inputs.base_branch }}" \
-            --head "$PR_BRANCH" \
+            --head "$PR_BRANCH" ${DRAFT_FLAG} \
             --title 'Merge `${{ inputs.head_branch }}` into `${{ inputs.base_branch }}`' \
-            --body '${{ inputs.pr_message }}' \
-            --draft
+            --body '${{ inputs.pr_message }}'
+
+          # Trigger tests is requested
+          if [[ "${{ inputs.create_as_draft }}" != "true" ]] ; then
+            PR_URL=$(gh pr view "${PR_BRANCH}" --json url | jq -r '.url')
+
+            gh pr comment "$PR_URL" --body "@swift-ci please test"
```

Comment:
nit: just `@swift-ci test` works

Reply:
I know, but let's encourage folks to be polite :)